mcs/class/corlib/Test/System/GetInterfaceTest.cs

```diff
@@ -0,0 +1,38 @@
+// GetInterfaceTest.cs - NUnit Test Cases for https://github.com/mono/mono/issues/6579
+//
+// Konstantin Khitrykh (konh@yandex.ru)
+//
+// (C) Konstantin Khitrykh
+// 
+
+using NUnit.Framework;
+using System;
+using System.Collections.Generic;
+
+namespace MonoTests.System {
+	[TestFixture]
+	public class GetInterfaceTest {
+
+		[Test]
+		public void GetInterfaceTests() {
```

Comment:
1. This probably should go in `mcs/class/corlib/Test/System/TypeTest.cs` we try to group tests by the class of the method that they're testing.
2. The name of the test should be a little more descriptive: `GetInterfaceCaseInsensitiveTest`